Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -147,9 +146,9 @@ public struct ChatMessage<CustomData: ChatCustomData>: Identifiable, Codable {
   
   // MARK: Dynamic Member Lookup
   
-  public subscript<T>(dynamicMember keyPath: WritableKeyPath<CustomData.Message, T>) -> T {
-    get { content.custom[keyPath: keyPath] }
-    set { content.custom[keyPath: keyPath] = newValue }
+  public subscript<T>(dynamicMember keyPath: WritableKeyPath<CustomData.Message, T?>) -> T? {
```

Comment:
By returning `T?`, if the type is an optional then you're actually giving the user a `Type??` which can be hard to make use of.

For example, if the custom object is
```
struct CustomMessage: MessageCustomData {
  var value: String? 
}
```

then accessing the `value` property (`message.value`) would be of type `String??` which can have type matching issues.